tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -446,6 +446,31 @@ var _ = Describe(
 				}
 			})
 		})
+
+		It("includes a specific NTP server among sources", reportxml.ID("74409"), func() {
+			By("Check if the target cluster is connected")
+			connected, err := cluster.Connected(APIClient)
+
```

Comment:
Add connection to check that MGMTConfig.AdditionalNTPSources is not empty

Reply:
Updated the initial value to empty and updated the skip condition.